src/raft/raw_node.rs

```diff
@@ -46,8 +46,8 @@ pub enum SnapshotStatus {
     Failure,
 }
 
-fn is_local_msg(m: &Message) -> bool {
-    match m.get_msg_type() {
+pub fn is_local_msg(t: MessageType) -> bool {
```

Comment:
No, tests in `tests` directory are integration tests, tests in same source file are unit tests, they are not the same thing.

We may figure out another way to reuse these common code rather than just putting all the tests to tests directory.